### PR TITLE
Several ADAS updates to 2017 CR-V dbc

### PR DESCRIPTION
**Pull request by energee**

### CODE REVIEW

File: honda_crv_2017_can.dbc [hunk 1]
```diff
@@ -2,255 +2,306 @@ VERSION ""
...
 
 BO_ 344 POWERTRAIN_DATA: 8 PCM
- SG_ TRANSMISSION_SPEED : 7|16@0+ (0.002759506,0) [0|70] "m/s"  NEO
- SG_ ENGINE_RPM : 23|16@0+ (1,0) [0|15000] "rpm"  NEO
- SG_ TRANSMISSION_SPEED2 : 39|16@0+ (0.002759506,0) [0|70] "m/s"  NEO
- SG_ COUNTER : 61|2@0+ (1,0) [0|3] ""  NEO
- SG_ CHECKSUM : 59|4@0+ (1,0) [0|3] ""  NEO
+ SG_ XMISSION_SPEED : 7|16@0+ (0.002759506,0) [0|70] "m/s" NEO
+ SG_ ENGINE_RPM : 23|16@0+ (1,0) [0|15000] "rpm" NEO
+ SG_ XMISSION_SPEED2 : 39|16@0+ (0.1723,0) [1|0] "" NEO
```
Comment (rbiasini) [inline honda_crv_2017_can.dbc]: What do you think the unit is here? not m/s anymore? What about the limits?

Comment (energee) [reply honda_crv_2017_can.dbc]: I'll revert that to what we have in the other Hondas, I was doing something else with it

**Comment by rbiasini** [comment]
@energee , sounds great. does this imply any required change to openpilot repo?

**Comment by energee** [comment]
No, I don't have any code in the comma repo yet for this car, but will be coming soon!

**Comment by energee** [comment]
2017 != 2016

**Comment by rbiasini** [comment]
@energee , right, this is for 2017. ACC should be just like the 2016+ Civic. 

I would just rename this dbc to honda_crv_touring_2017_can.dbc to be consistent with other dbc file names, other than that it looks good to me.

**Comment by energee** [comment]
I don't have the touring trim, i have the EX, should I still rename it?

**Comment by rbiasini** [comment]
yes. It's better to have the trim in the dbc file name. Let's rename it as honda_crv_ex_2017_can.dbc
For the near future, I'm thinking about a cleaner dbc file name convention... I'll accept suggestions on Slack

**Pull request merged**